src/site/content/en/blog/multi-origin-pwas/index.md

```diff
@@ -96,20 +94,13 @@ Since there's no way to share permissions across origins, the only solution here
 To install a PWA, each origin must have its own manifest with a `start_url` that's [relative to itself](https://w3c.github.io/manifest/#start_url-member). This means that a user receiving the installation prompt on a given origin (i.e: `https://section.example.com`) won't be able to install the PWA with a `start_url` on a different one (i.e: `https://www.example.com`).
 In other words, users receiving the installation prompt in a subdomain will only be able to install PWAs for the subpages, not for the main URL of the app.
 
-A simple workaround is to use a`start_url` with a redirect to the main origin. For example: subdomain `https://section.example.com`, can define a `start_url` of `https://section.example.com/pwa`, containing a redirect to: `https://www.example.com`, making the user always land at the home page.
-
 There's also the issue that the same user could receive multiple installation prompts when navigating the site, if each subdomain meets the [installation criteria](https://developers.google.com/web/fundamentals/app-install-banners/#criteria), and prompts the user to install the PWA.
 
-Mitigate this problem by applying any of the following techniques:
-
-- **Showing the prompt on a single domain:** If the site has many subdomains that pass the installation criteria, the `beforeinstallprompt` event could be used and `preventDefault()` called (as explained [here](https://developers.google.com/web/fundamentals/app-install-banners/#listen_for_beforeinstallprompt)), to prevent the prompt from appearing in unintended parts of the site (like subdomains), while continue to show it in other parts (e.g. the home page).
-
-- **Showing the prompt once, across different subdomains:** It's also possible to let each subdomain show the installation prompt. (This can make sense if some subdomains are more frequently visited). In this case, it's important to avoid showing the prompt to a user multiple times. To that purpose, a cookie could be used, to track if the prompt has already been shown, and check for its existence when the `beforeinstallprompt` event is triggered. If the cookie is present, it means that the user already received the prompt somewhere else, so `preventDefault()` can be called, to avoid showing it again.
-
+To mitigate this problem you can make sure that the prompt is shown only on a single domain. If the site has many subdomains that pass the installation criteria, the `beforeinstallprompt` event could be used and `preventDefault()` called (as explained [here](https://developers.google.com/web/fundamentals/app-install-banners/#listen_for_beforeinstallprompt)), to prevent the prompt from appearing in unintended parts of the site (like subdomains), while continue to show it in other parts (e.g. the home page).
```

Comment:
@demianrenzulli I see a few ways this paragraph can be improved. I'm going to give you guidelines but leave it up as an exercise for you to try.

1. Rephrase the link text following these directions: https://developers.google.com/style/link-text
1. Rephrase this section to use the [active voice](https://developers.google.com/style/voice): `the beforeinstallprompt event could be used and preventDefault() called`
1. Convert this section into a numbered list. I'll give you a hint on how it should start:

```
To mitigate this problem, make sure that the prompt is shown only on a single domain:

1. ...
```

Reply:
Looks great! Good job @demianrenzulli.